### DISCUSSION TITLE
VOD stopped uploading to S3 buckets

### DISCUSSION THREAD

**Discussion by wbenarto** [ant-media/Ant-Media-Server#5909] [Q&A]
Hello, I'm having some issues with uploading VODs to S3 bucket and hoping to get some clue here. 

My WebRTCAppEE is set up to record as MP4 and S3 recording. Everything was working fine until in Nov 2023 we were unable to see any recording in our S3. I have checked with the S3 configuration and found nothing wrong. Please see image attached: 
![antmedia](https://github.com/ant-media/Ant-Media-Server/assets/42128969/b1385bfb-6c83-42b4-8d69-4adb087cfb2e)

Here's some errors logged: 
![image](https://github.com/ant-media/Ant-Media-Server/assets/42128969/bb97c8aa-08f7-49af-abfb-a322fa66911b)
![image (1)](https://github.com/ant-media/Ant-Media-Server/assets/42128969/c7a4b37a-7d0b-44f6-8334-cc4492bf6392)

When both MP4 and S3 recording boxes are checked we are unable to view the VOD and S3 are not uploaded, but when only MP4 box is checked VOD is back to normal. We tried to upload via AWS CLI on the same bucket to the same directory and it worked.

Please let me know what could possibly the cause. Thank you!

**Comment by Mohit-3196** [comment]
Hi @wbenarto,
How are you?

It looks to be related to the S3 parameters/settings primarily.
Please tell me which AMS version you are using so I can try the S3 uploads at my end?

Thank you.

**Reply by wbenarto** [reply]
@Mohit-3196 I'm doing really well thanks! How about you? 

So sorry, where can I get AMS version information for you?

**Reply by lastpeony** [reply]
Login to web panel and its written on bottom
